### DISCUSSION TITLE
Can we request to add Starkey wallet in explorer.aptoslabs.com?

### DISCUSSION THREAD

**Discussion by ghost** [aptos-labs/aptos-developer-discussions#370] [General]
Hi Aptos Team,

I have been using the Starkey wallet (https://www.starkey.app) for a MOVE Project where we are using APTOS. However, I noticed no option to login via the Starkey wallet in the explorer.

Kindly Please add the Starkey wallet to the login page alongside the other wallets.

![Screenshot 2024-08-19 233249](https://github.com/user-attachments/assets/0f229cfc-f192-4424-9b2a-b9432e7146dc)
![Screenshot 2024-08-19 233257](https://github.com/user-attachments/assets/fc937930-f2dd-4e36-beac-e4d9021e9332)

**Comment by 0xmaayan** [comment]
Hi @Jatinpandya26 , all wallets displayed on Explorer are integrated with the [Aptos Wallet Adapter](https://github.com/aptos-labs/aptos-wallet-adapter), to support a new wallet - the wallet team should integrate with the adapter by following the [integration guide](https://aptos.dev/en/build/sdks/wallet-adapter/browser-extension-wallets)

**Reply by ghost** [reply]
I will ask the Wallet team to do the same, Thanks, @0xmaayan
